src/service-worker/install.js

```diff
@@ -36,8 +35,7 @@ export function installCacheServiceWorker(win) {
     if (!('serviceWorker' in navigator)) {
       return;
     }
-    if (!getMode().localDev &&
-        win.location.hostname !== parseUrl(urls.cdn).hostname) {
+    if (!getMode().localDev && !isProxyOrigin(win.location.hostname)) {
```

Comment:
This one is a special case. We cannot install the serviceworker from `cdn.ampproject.org` if we're on one of the new proxy subdomains.